test/e2e/multikueue/e2e_test.go

```diff
@@ -587,6 +588,7 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 					Image(util.E2eTestAgnHostImage, util.BehaviorWaitForDeletion). // Give it the time to be observed Active in the live status update step.
 					Parallelism(2).
 					RequestAndLimit(corev1.ResourceCPU, "1").
+					TerminationGracePeriod(1).
```

Comment:
I think we cannot do this here, as we later expect the Workload to be successful: https://github.com/kubernetes-sigs/kueue/pull/4660/files#diff-5dcb842cd0d9d77809e502564dd2fa3e151d3ad48945f7075dc6aeeed483852bR624

This would risk flaking again.

Reply:
Ah, maybe we can, because we call `WaitForActivePodsAndTerminate`, and so we terminate the pods without deleting actually.